pkg/plugins/resources/postgres/store.go

```diff
@@ -316,3 +346,39 @@ func (r *resourceMetaObject) GetCreationTime() time.Time {
 func (r *resourceMetaObject) GetModificationTime() time.Time {
 	return r.ModificationTime
 }
+
+func registerAndCollectMetrics(pc core_plugins.PluginContext, dbStats sql.DBStats) error {
+
+	if err := pc.Metrics().Register(postgresConnectionMetric); err != nil {
+		return err
+	}
+
+	if err := pc.Metrics().Register(postgresMaxConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresClosedConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresWaitConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresWaitConnectionDurationMetric); err != nil {
+		return err
+	}
```

Comment:
@jakubdyszkiewicz , I thought of using `MustRegister` to register all collectors at the same time and look for panic. Is that okay?

Reply:
I'd rather use explicit error